pom.xml

```diff
@@ -440,6 +441,51 @@
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>jira</artifactId>
+            <version>2.3</version>
+            <exclusions>
```

Comment:
@vivek that change was merged. When I get https://github.com/jenkinsci/jira-plugin/pull/127 merged I will ask the maintainers to release a new version.